### DISCUSSION TITLE
Percentage and remaining time in 1.3.1

### DISCUSSION THREAD

**Discussion by ghost** [Jyers/Marlin#540] [Q&A] [answered]
It seems that Percentage and remaining time in 1.3.1 are always on zero
![IMG_20210419_141944.jpg](https://user-images.githubusercontent.com/39264449/115284404-a5558900-a11a-11eb-8c09-27fb195ed160.jpg)

Is it a known issue?

**Comment by Cal-LF** [comment]
Mine still shows 99% at end of print.

**Comment by Jyers** [comment, marked as answer]
I now realized that I only fixed this for host prints, I'll try to fix this for sd prints soon. Also if both are zero on a host print, you may have your plugins incorrectly configured.

**Comment by aryajuanda** [comment]
Do you use M73 progress plugin.. host print is working great for me.. 

Have not check the sd print.. never use it anymore since using this firmware.. always start from octoprint or octopod.. and use LCD to cancel/stop if failed on 1st layer..
